### PR TITLE
Adds new documentation for Alnum validator

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | yes

### CODE REVIEW

File: docs/book/validators/alnum.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Alnum
+
+`Laminas\I18nn\Validator\Alnum` allows you to validate if a given value
+**contains only alphabetical characters and digits**. There is no length
+limitation for the input you want to validate.
+
+## Basic Usage
+
+```php
+$validator = new Laminas\I18nn\Validator\Alnum();
```
Comment (michalbundyra) [inline docs/book/validators/alnum.md]: typo: `I18n` (one `n`) - please check all other places as I can see it multiple times in that file

File: docs/book/validators/alnum.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+if ($validator->isValid('Abcd12')) {
```
Comment (michalbundyra) [inline docs/book/validators/alnum.md]: The same about examples as in previous review.

File: docs/book/validators/alnum.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Alnum
+
+`Laminas\I18nn\Validator\Alnum` allows you to validate if a given value
+**contains only alphabetical characters and digits**. There is no length
+limitation for the input you want to validate.
```
Comment (michalbundyra) [inline docs/book/validators/alnum.md]: We might want add a note "if you need length validator ... see... " and maybe some reference also to chains ?

File: docs/book/validators/alnum.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+By default, whitespace is not accepted as it is not part of the alphabet.
+However, if you want to validate complete sentences or phrases, you may need to
+allow whitespace; this can be done via the `allowWhiteSpace` option, either at
+instantiation or afterwards via the `setAllowWhiteSpace()` method.  
+
+```php fct_label="Constructor Usage"
+$validator = new Laminas\I18nn\Validator\Alnum(['allowWhiteSpace' => true]);
+
+if ($validator->isValid('Abcd and 12')) {
```
Comment (michalbundyra) [inline docs/book/validators/alnum.md]: maybe better example, like: `5 apples`, `12th chapter` etc.

Comment (froschdesign) [reply docs/book/validators/alnum.md]: Right now, I would wait for [new tutorials](https://github.com/laminas/technical-steering-committee/pull/7#issue-389144985) and then we can update the content.
I hope that we can achieve consistency and reuse of the same example across the documentation.

**Comment by michalbundyra** [comment]
Thanks, @froschdesign!

**Pull request closed without merging**